bstrp_functions.sh

```diff
@@ -445,6 +446,7 @@ read_feature_options(){
     x) ToggleFeature AZURE_ENABLED ;;
     y) ToggleFeature SYSTEMD_ENABLED ;;
     z) ToggleFeature NANOFI_ENABLED ;;
+    aa) ToggleFeature SPLUNK_ENABLED ;;
```

Comment:
we should update the usage message printed (`Enter choice [ A - Z or 1-7 ]`) to reflect the addition of `aa` option

Reply:
I am not sure what would be the correct way to phrase this...
How about this? https://github.com/apache/nifi-minifi-cpp/pull/1219/commits/7a0401fdaf024ac39ec0b6f5cd2527c5c98762b4